### DISCUSSION TITLE
Welcome to Vercel Discussions!

### DISCUSSION THREAD

**Discussion by AndrewBarba** [swift-cloud/Vercel#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nonfungibletunji** [comment]
I am here, thanks. when we are developing locally and can look into having some sort of watcher for code changes, so we don't have to shut and restart the local server for every code edit change.
